### ISSUE TITLE
Header Overlaps Content on Scroll - Needs Fixed Positioning

### ISSUE THREAD

**Issue by saadgibawa** [Anjaliavv51/Retro#4]
![Screenshot 2024-10-01 200621](https://github.com/user-attachments/assets/0b560572-a16a-4e10-965a-15e824fda332)

I've noticed an issue with the header component on the website. When scrolling down, the header overlaps with the main content, making it hard to read and interact with the page. Additionally, the header scrolls along with the content, which affects the usability of the navigation. I want to work on this and fix it. So please assign me this issue @Anjaliavv51

**Comment by Vd7905** [comment]
i can fix this positioning issue of the header please assign this to me

**Comment by amanver45** [comment]
hey @Anjaliavv51 @saadgibawa , I'd like to contribute in this issue.

**Comment by Anjaliavv51** [comment]
@saadgibawa go ahead!

**Comment by Anjaliavv51** [comment]
@saadgibawa please check the pin issue #11 before starting to contribute.

**Comment by saadgibawa** [comment]
@Anjaliavv51 sure

**Comment by Anjaliavv51** [comment]
@amanver45 please create an issue , then i will assign u , as this issue is already assigned please , raise another issue.

**Comment by laxmikandivalasa** [comment]
![Screenshot 2024-10-01 233633](https://github.com/user-attachments/assets/9b728d41-5702-4b3b-9f4a-0dabfc4057de)
@Anjaliavv51 Here in the homepage seems to have Alignment Issues and uneven spaces in nav bar
please assign it to me...I'd like to contribute!!
